pkg/tidb/forwarder.go

```diff
@@ -118,6 +113,14 @@ func (f *Forwarder) pollingForTiDB() {
 	}
 }
 
+func (f *Forwarder) getEndpointAddr(port int) (string, error) {
+	if f.statusProxy.noAliveRemote.Load() {
+		log.Warn("Unable to resolve connection address since no alive TiDB instance")
```

Comment:
@HunDunDM PTAL about this line to see whether it will affect PD cluster without TiDB.